### ISSUE TITLE
some rules not working

### ISSUE THREAD

**Issue by plakhin** [vuejs/eslint-plugin-vue#235]
**Tell us about your environment**

* **ESLint Version:** 4.11.0
* **eslint-plugin-vue Version:** 3.14.0
* **Node Version:** 8.9.1

**Please show your full configuration:**

```
{
    "extends": [
        "eslint:recommended",
        "plugin:vue/recommended"
    ]
}
```

**What did you do? Please include the actual source code causing the issue.**

```vue
<template>
    <div>
        <ul>
            <li v-for="item in items">{{ item }}</li>
        </ul>
        <ul>
            <li v-for="item in items" :key="item" key="none">{{ item }}</li>
        </ul>
    </div>
    <div></div>
</template>
<script>
export default {
    data(){
        return {
            items: ['foo', 'bar', 'baz']
        }
    },

    props: ['items'],

    created(){
        this.items.map(item => {});
    },


}
</script>
<style scoped>
    ul {
        margin-top: 3rem;
    }
</style>
```

**What did you expect to happen?**
I expected to also get errors for no-dupe-keys and no-duplicate-attributes

**What actually happened? Please include the actual, raw output from ESLint.**
```
eslint ./test.vue

./test.vue
   4:13  error  Elements in iteration expect to have 'v-bind:key' directives  vue/require-v-for-key
  10:5   error  The template root requires exactly one element                vue/valid-template-root
  23:24  error  'item' is defined but never used                              no-unused-vars

✖ 3 problems (3 errors, 0 warnings)
```

**Comment by michalsnik** [comment]
Hey. Thanks for posting this issue, however what you’re expiriencing is expected. Take a look at documentation of your version: https://github.com/vuejs/eslint-plugin-vue/tree/v3.14.0
Those rules are not enabled by default. You have to manually turn them on :)

Be aware that current master is far ahead last released version as we’re getting ready for first official release :)

**Comment by plakhin** [comment]
Yes, it is not obvious that I should look for v3.14.0 tree as it is not mentioned anywhere in the master branch readme. Also, I thought that vue/recommended is in sync with "Priority C Rules: Recommended" of official style guide. Anyway, thank you for response and of course for all the great amount of work you did.
@michalsnik
